spring-cloud-function-samples/function-sample-azure/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM springcloudstream/azure-functions-java17:1.0.0
```

Comment:
Note that I uploaded a hand built base Java 17 image in lieu of this getting merged into Azure https://github.com/Azure/azure-functions-docker/pull/667